euler/euler.ipynb

```diff
@@ -1,12 +1,22 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Euler\n",
+    "Written by Egbert Ammicht<br>\n",
+    "Created: November 16, 2018<br>\n",
+    "Last updated: June 30, 2020"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Panel dashboard illustrating Euler's Method\n",
     "\n",
-    "Euler's method is a numerical integration technique for solving ordinary differential equations. Specifically, given $y' = f( x, y )$ and a starting point $(x_0, y_0)$, Euler's method provides approximate values of $y$ for $x>x_0$. To explore how this method works, this notebook from [ea42gh](https://github.com/ea42gh) sets up a dashboard using [Panel](http://panel.pyviz.org), [HoloViews](http://holoviews.org), and [Bokeh](http://bokeh.pydata.org). You will need to have a live Python process running (not just an Anaconda.org notebook or web page), which you can set up using `conda install -c pyviz/label/dev panel holoviews bokeh`."
+    "Euler's method is a numerical integration technique for solving ordinary differential equations. Specifically, given $y' = f( x, y )$ and a starting point $(x_0, y_0)$, Euler's method provides approximate values of $y$ for $x>x_0$. To explore how this method works, this notebook from [Egbert Ammicht](https://github.com/ea42gh) sets up a dashboard using [Panel](http://panel.pyviz.org), [HoloViews](http://holoviews.org), and [Bokeh](http://bokeh.pydata.org). You will need to have a live Python process running (not just an Anaconda.org notebook or web page), which you can set up using `conda install -c pyviz/label/dev panel holoviews bokeh`."
```

Comment:
Probably don't need label/dev anymore?

Reply:
Thanks for pointing that out - updated in latest commit